docs/guides/addresses.mdx

```diff
@@ -44,29 +44,36 @@ An `address` is a feature type that represents a physical place through a series
 | Country | Address Count |
```

Comment:
Just my opinion so feel free to ignore, but I think this list would be better ordered by country rather than count.

Reply:
Good call, that makes more sense than ordering by count.. this should be updated in the most recent commit.